code/go/0chain.net/chaincore/transaction/entity.go

```diff
@@ -412,10 +412,7 @@ func (t *Transaction) GetSummary() *TransactionSummary {
 - applicable only when running in test mode and the transaction_data string contains debug keyword somewhere in it
 */
 func (t *Transaction) DebugTxn() bool {
-	if !config.Development() {
```

Comment:
this will be fixed in different PR, check this [[Logging] Add trace level](https://github.com/0chain/0chain/issues/1636)

Reply:
Ah okay, I'll revert it